dashboard/lib/sample_data.rb

```diff
@@ -0,0 +1,181 @@
+include FactoryGirl::Syntax::Methods
+
+class SampleData
+  TEST_ACCOUNT_PASSWORD = "00secret".freeze
+  @@prng = nil
+
+  # Returns a seeded random number generator for consistent test data
+  def self.prng
```

Comment:
What is the p? Persistent? 
Is this a common abbreviation? I'd prefer a more clear and descriptive name.

Reply:
Seems to be a common abbreviation for [Pseudorandom Number Generator](https://en.wikipedia.org/wiki/Pseudorandom_number_generator).

Reply:
guess I should have googled that. How about changing the comment `s/random/pseudorandom` ?

Reply:
I had chosen `prng` also because Ruby docs use that variable name in their samples for `Random`: https://ruby-doc.org/core-2.2.0/Random.html

But it may be unnecessarily unconfusing. Changed to `rng`.